python/mhs/cf_common.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
```

Comment:
Shebangs should not be in files that do not have a `__main__` since these are not script and are not meant to be executed directly. This comment may apply also to other files, please review the `.py` files.